### ISSUE TITLE
bootstrap.sh fails on Raspbian

### ISSUE THREAD

**Issue by blitzcode** [haskell/cabal#3400]
Here:

   Downloading Cabal-1.25.0.0...
     % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                    Dload  Upload   Total   Spent    Left  Speed
     0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (22) The requested URL returned error: 404 Not Found

   Error during cabal-install bootstrap:
   Failed to download Cabal.

**Comment by 23Skidoo** [comment]
This is a known issue, the bootstrap script currently only works for released versions. So you can either use `cabal-install-1.24.0.0.tar.gz` from Hackage, or do

```
$ cd Cabal
$ runghc Setup.hs configure
$ runghc Setup.hs build
$ runghc Setup.hs install
$ cd ../cabal-install
$ sh bootstrap.sh 
```

**Comment by 23Skidoo** [comment]
Fixed in #3439.
